### PR TITLE
add sync of user courses

### PR DESCRIPTION
**Pull request by inlanger**
https://jira.10clouds.com/browse/ED-459

### CODE REVIEW

File: openedx/core/djangoapps/content/course_overviews/connector.py [hunk 1]
```diff
@@ -93,3 +93,38 @@ def update_or_create_verification_course(self,
...
+        course_ptr_id = self.get_course(openedx_course_id)
+        if self.check_user_course(course_ptr_id):
+            self.cursor.execute("""INSERT INTO courses_courseuser
+                                   (state, removed, course_ptr_id, student_id)
+                                   VALUES ('undergoing', '0', '{}', '{}');
+                                """.format(course_ptr_id, edevate_user)
+                                )
+            self.connection.commit()
+            return True
+        return False
```
Comment (CrowbarKZ) [inline openedx/core/djangoapps/content/course_overviews/connector.py]: this return values are not used anywhere so probably we can remove them

File: common/djangoapps/student/views.py [hunk 2]
```diff
@@ -985,6 +986,13 @@ def _credit_statuses(user, course_enrollments):
     return statuses
 
 
+def edevate_update_user_course_list(user, course_id):
+    edevate_db = EdevateDbConnector()
+    edevate_db.update_users_course_list(course_id, user.email)
+    edevate_db.close()
+    return None
```
Comment (CrowbarKZ) [inline common/djangoapps/student/views.py]: not needed, it by default returns None

**Comment by CrowbarKZ** [comment]
LGTM. But probably will need to be changed to some API way instead of directly connecting to the other database.

**Comment by inlanger** [comment]
For this release I've made directly connection to db, next release we will update with API

**Comment by aniav** [comment]
LGTM

**Pull request merged**